README.md

```diff
@@ -574,15 +574,21 @@ To take control over the inner organization of each field (each form row), you c
 A field template is basically a React stateless component being passed field-related props so you can structure your form row as you like:
 
 ```jsx
+
+import {Help, ErrorList} from "react-jsonschema-form/lib/components/fields/SchemaField"; // You can import also the default 'Label' from SchemaField. Below we use a custom label.
```

Comment:
That would solve the issue with people loading the lib through from a script tag:

```js
 <script src="https://unpkg.com/react-jsonschema-form/dist/react-jsonschema-form.js"></script>
``` 

There you have no access to individual modules, and I'm uncomfortable attaching them to the root exposed object or anything like that.